phpunit-full.xml.dist

```diff
@@ -15,7 +15,7 @@
     </php>
 
     <testsuite name="API Test Suite">
-        <directory>Tests</directory>
+        <directory>Tests/Controller</directory>
```

Comment:
takže na jekinsi pojede pouze toho? nechybí tam ještě `Tests/Command`?

Reply:
jj, to na travisu nepojede, protoze to potrebuje symfony kernel a ten zas potrebuje syrup
Tests/command chybi! doplnil jsem